tools/mason/MasonExternal.chpl

```diff
@@ -199,7 +199,7 @@ private proc generateYAML() {
     remove(yamlFilePath);
   }
   const reposOverride = 'repos:\n'+
-                        '  - ~/.mason/spack-registry/var/spack/repos/builtin \n';
+                        '  - ' + MASON_HOME + '/spack-registry/var/spack/repos/builtin \n';
```

Comment:
Just so you are aware, you can also do this:

```suggestion
                        '  - %s/spack-registry/var/spack/repos/builtin \n'.format(MASON_HOME);
```

Either form works fine though.

Reply:
Thanks, I'll take care next time :)